doc/source/guide/acquiring_data/sample-data.rst

```diff
@@ -11,7 +11,8 @@ download the sample data simply run the following command::
 
     import sunpy.data
     sunpy.data.download_sample_data()
-
+With a bad Internet connection it helps to set the overwrite=False parameter
```

Comment:
If you look at the [Travis errors](https://travis-ci.org/sunpy/sunpy/jobs/158358387#L1858), you find the following:
```
sample-data.rst:14: WARNING: Literal block ends without a blank line; unexpected unindent.
```